client_react/src/components/gamePage/CountdownTimer.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import moment from 'moment';
+
+export default function ({ deadline, votingFinished }) {
+    const isTimerExpired = total => total < 0;
+    const [timer, setTimer] = useState(getTimeRemaining(deadline));
+
+    setTimeout(() => {
+        const timer = getTimeRemaining(deadline);
+        isTimerExpired(timer.total) ? votingFinished() : setTimer(timer)
+    }, 1000);
+
+    return !isTimerExpired(timer.total)
+        ? (<div>
+            {`${timer.hours} hours `}
+            {`${timer.minutes} minutes `}
+            {`${timer.seconds} seconds `}
+        </div>)
+        : <div>Voting is closed</div>
+};
+
+
+
+function getTimeRemaining(deadline) {
+    const localOffset = new Date().getTimezoneOffset();
+    const total = moment(deadline)
+        .subtract(localOffset, 'minutes')
+        .diff(moment(), 'milliseconds');
+    const seconds = Math.floor((total / 1000) % 60);
+    const minutes = Math.floor((total / 1000 / 60) % 60);
+    const hours = Math.floor((total / (1000 * 60 * 60)));
+    ///const days = Math.floor(total / (1000 * 60 * 60 * 24));
```

Comment:
Why days are commented?

Reply:
I decided to follow such pattern:  hours:minutes:seconds
So if there are more than 1 day left for voting: timer will show (for example) 35 hours 2 minutes 56 seconds

Reply:
Ok then, looks good